src/test/resources/features/signup.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Sign Up
+
+  Background:
+    Given that I enter the conduit site
+    And I click on the Sign Up link
+    And the system shows me the registration form
+
+  @SuccessfulSignUp
+  Scenario: Successful registration of a new user
+    When I filled out the fields with valid values
+    And I click the sign up button
+    Then the system displays a successful registration message
+
+  @EmptyFields
+  Scenario: Empty sing up fields
+    When I leave the form fields blank
+    And I click the sign up button
+    Then the system displays error messages corresponding to blank fields
+
+  @ExistingEmail
+  Scenario Outline: Existing email
+    When I fill in the username and password fields with valid data
+    And I fill out the email field with mail "<email>"
+    And I click the sign up button
+    Then the system displays a message indicating that the mail already exists
+
+    Examples:
+      | email                      |
+      | anamaria.quintero@wolox.co |
+
+  @ExistingUser
+  Scenario Outline: Existing user
+    When I filled in the fields with existing values "<username>"
+    And I click the sign up button
+    Then the system displays the error message indicating that the person already exists
+    Examples:
+      | username |
+      | amquinte |
+
+  @InvalidFormat
+  Scenario Outline: Invalid Format email
+    When I fill in the username and password fields with valid data
+    And I fill out the email field with mail "<email>"
+    And I click the sign up button
+    Then The email field is in invalid state
+    And The system displays an error message in the email
+
+    Examples:
+      | email                     |
+      | anamaria.quinterowolox.co |
+      #| anamaria.quintero@wolox |
```

Comment:
Remove this comment

Reply:
That commented line is for testing without the .com or .com ... etc. I left it commented because the application is not doing this validation. I will uncomment that line and adjust the scenario to send the expected error messages from there